### ISSUE TITLE
Cannot run osxphotos after upgrading to pipx 1.1.0

### ISSUE THREAD

**Issue by petebocken** [RhetTbull/osxphotos#712]
osxphotos v0.50.3

June 3 I upgraded pipx 1.0.0 -> 1.1.0 and ever since when I run osxphotos, I get:
```
zsh: /Users/peter/.local/pipx/venvs/osxphotos/bin/osxphotos: bad interpreter: /Users/peter/.local/pipx/venvs/osxphotos/bin/python: no such file or directory
```

I was going to try downgrading pipx, but when I run `brew install pipx@1.0.0` It just says `Warning: No available formula with the name "pipx@1.0.0".`  So not sure what I'm doing wrong there.

Any advice?

**Comment by petebocken** [comment]
Guess I figured it out. I had to update my symlinks for `/Users/peter/.local/pipx/venvs/osxphotos/bin/python3.10` to point to `/opt/homebrew/Cellar/pipx/1.1.0/libexec/bin/python3.10`.

It was still pointed to `/opt/homebrew/Cellar/pipx/1.0.0/libexec/bin/python3.10`

Not sure why that didn't update when I updated to pipx 1.1.0.

When I run osxphotos, it does give this warning now (assuming this will be addressed at some point?).
`2022-06-14 19:57:38,800 - WARNING - photosdb.py - 112 - WARNING: This module has only been tested with macOS versions [10.12, 10.13, 10.14, 10.15, 10.16, 11.0, 11.1, 11.2, 11.3, 11.4, 11.5, 11.6, 12.0, 12.1, 12.2, 12.3]: you have Darwin, OS version: 12.4`
